### DISCUSSION TITLE
TensorFlow backend not working

### DISCUSSION THREAD

**Discussion by nirmal129** [lululxvi/deepxde#1245] [Q&A] [answered]
Hello, 

I am working on Colab and want to use tf backend, however, DeepXDE is unable to find it. My code is shown below. 

`%pip install deepxde --quiet`
`import deepxde as dde`
`from deepxde.backend import tf`

And I am getting the error shown below. 

<img width="1111" alt="Screen Shot 2023-04-13 at 5 16 47 PM" src="https://user-images.githubusercontent.com/61615966/231895310-a3f2a3a2-e893-44c1-8b91-8aef16076540.png">

It was working yesterday but stopped working today. Can someone help me out? 

Thank you!

**Comment by lululxvi** [comment, marked as answer]
As indicated in the first line, you didn't select a backend. Previously, DeepXDE will use tf v1 as default, but DeepXDE 1.8.3 removed this default behavior.

The new version DeepXDE 1.8.4 will have a better way to set the default backend. Coming very soon.

**Comment by nirmal129** [comment]
How do I select a backend on Colab?

**Comment by lululxvi** [comment]
DeepXDE 1.8.4 is published.
